### PR TITLE
Home page HTML integration

### PR DESCRIPTION
**Pull request by techaffinity**
We have applied HTML intgration for some of the pages. This build includes some commented code as well as dummy text without translation. We will be updating this on knowing the usage of commented code and check for the translation if we get the real content placing there.

### CODE REVIEW

**Comment by smn** [comment]
Merged `develop` into this and resolved conflicts.

**Comment by techaffinity** [comment]
We have updated the poll feature html integration to this branch. As discussed with Bruce, We have kept poll results choices to have 3 different colors. The choices more than 3 will follow the three colors theme again in a sequential order. This has been implemented as per the given html theme. We have also modified some CSS related to registration module.

File: polls/views.py [hunk 1]
```diff
@@ -21,9 +21,27 @@ class DetailView(generic.DetailView):
...
+        choice.percentage = vote_percentage
+        choice.color = choice_color[index]
+        index += 1
+
+    context = {
+        'question': question,
+        'total': total_votes,
+        'choices': choices
+    }
+    return render(request, 'polls/results.html', context,)
```
Comment (smn) [inline polls/views.py:44]: Please add a test for this.

**Comment by smn** [comment]
Please go ahead and merge so we can get this automatically deployed to the QA server and get a sense of what it looks like. Changes can happen in follow up pull-requests that will then be reviewed. Smaller PRs are easier.

**Pull request merged**